src/router/index.js

```diff
@@ -21,6 +23,16 @@ const routes = [
     name: 'Profile',
     // add component later
   },
+  {
+    path: '/AdminActiveRes',
+    name: 'AdminActiveRes',
+    component: AdminActive, // imported component
```

Comment:
Comment not necessary
```suggestion
    component: AdminActive, 
```